test/unit/api/ledgerAccountTypes/handler.test.js

```diff
@@ -37,6 +38,11 @@ Test('LedgerAccountTypes', ledgerAccountTypesHandlerTest => {
     sandbox = Sinon.createSandbox()
     sandbox.stub(Logger)
     sandbox.stub(LedgerAccountTypeService)
+    sandbox.stub(ProxyCache, 'getCache').returns({
+      connect: sandbox.stub(),
+      disconnect: sandbox.stub(),
+      healthCheck: sandbox.stub().returns(Promise.resolve())
```

Comment:
the same, but with less code )
```
 ...
 healthCheck: sandbox.stub().resolves()
```

Reply:
I'll clean this up in another PR. Copilot auto-complete makes me skip over sometimes.

Reply:
nvm. done. cleaned up here and throughout so copilot makes better suggestion next time.